test/integration/template_test.rb

```diff
@@ -211,4 +211,19 @@ def test_exception_handler_does_reraise_if_it_returns_true
     end
     assert exception.is_a?(Liquid::ZeroDivisionError)
   end
+
+  def test_global_filter_option_on_render
```

Comment:
These tests you added to TemplateTest are almost the same as the ones you added to VariableTest.  I don't know which is the best file to add the tests, but keep the tests in one of them and remove the additional tests in the other.